.github/workflows/contract-testing.yml

```diff
@@ -62,5 +65,49 @@ jobs:
        cat inputs/inputs_1_create.json
        cat inputs/inputs_1_update.json
        
+       make run-contract-testing
+       make delete-test-resources
+  auditing:
+    needs: change-detection
+    if: ${{ needs.change-detection.outputs.auditing == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: setup Atlas CLI
+      uses: andreaangiolillo/atlas-cli-github-action@v1.1.1
```

Comment:
is this correct?

Reply:
yes, it is using https://github.com/andreaangiolillo/mongodb-atlas-cli-github-action